datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -2225,39 +2285,6 @@ mod test {
         Ok(())
     }
 
-    #[test]
-    fn custom_type_literal_extensions() -> Result<()> {
```

Comment:
As we no longer produce user-defined types for any scalars or types, this test and the one below don't work.